### ISSUE TITLE
use client on windows

### ISSUE THREAD

**Issue by BlackCoyote** [rivafarabi/deckboard#268]
I would like to be able to use the client app on a windows laptop. Is there any way to do this without an android emulator?

**Comment by Moinax** [comment]
Yeah I'm working with a Windows Surface, and it would be awesome if I can use it as a client

**Comment by darkhog** [comment]
Have you tried Android emulators, such as Bluestacks?

**Comment by BlackCoyote** [comment]
> Have you tried Android emulators, such as Bluestacks?

I have not, but I would suspect that that would work. You might have missed though that this issue explicitly requests for this to be possible without any android emulators. It's just a lot of hassle and far from ideal if it requires an emulator.

**Comment by darkhog** [comment]
And I agree there should be a windows client. I was merely proposing a workaround for the time being.   -- Dariusz &#34;Darkhog&#34; G. Jagielski  
            
          
     
      
       
        Dnia 27 grudnia 2019 19:15 BlackCoyote &lt;notifications@github.com&gt; napisał(a):
       
    
       
         
 Have you tried Android emulators, such as Bluestacks? 
 
 I have not, but I would suspect that that would work. You might have missed though that this issue explicitly requests for this to be possible without any android emulators. It&#39;s just a lot of hassle and far from ideal if it requires an emulator.
